frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwarePackageCard/SoftwarePackageCard.tsx

```diff
@@ -148,6 +161,7 @@ const PackageStatusCount = ({
           {count} hosts
         </a>
       }
+      className={`${baseClass}__status`}
```

Comment:
Stylistically can we move className up to line 147?

Reply:
it's a prop on `DataSet` not on `TooltipWrapper`, so I'll move up to 145